src/libraries/System.Runtime.Serialization.Schema/tests/System/Runtime/Serialization/Schema/Import/ImporterTests.cs

```diff
@@ -112,13 +112,13 @@ public static IEnumerable<object[]> Import_MemberData()
             int newlineSize = Environment.NewLine.Length;
 
             // Import(XmlSchemaSet)
-            yield return new object[] { (XsdDataContractImporter imp) => imp.Import(SchemaUtils.PositiveSchemas), 5060 + (168 * newlineSize) };   // 168 lines
```

Comment:
FYI, this file has been updated in preparation for future serializer development. (#104173) The same approach to fixing applies (same 3 updates here actually) but there are a few additional lines to update the character count on.

Reply:
@StephenMolloy can you please take care of that?

Reply:
Done